drakrun/drakrun/draksetup.py

```diff
@@ -83,6 +84,81 @@ def check_root():
         return True
 
 
+def sanity_check():
+    logging.info("Checking xen-detect...")
+    proc = subprocess.run('xen-detect -N', shell=True)
+
+    if proc.returncode != 1:
+        logging.error('It looks like the system is not running on Xen. Please reboot your machine into Xen hypervisor.')
+        return False
+
+    logging.info("Testing if xl tool is sane...")
+
+    try:
+        subprocess.check_output('xl info', shell=True, stderr=subprocess.STDOUT)
```

Comment:
By default `subprocess.run` will print the output to the user, which is not intended in case of commands like `xl info`. Should we `stdout=subprocess.PIPE` and `stderr=subprocess.PIPE` then?

Reply:
if we want to silence the stdout IMO the best way would be to `stdout=subprocess.DEVNULL`, but it we want to process the output later `capture_output=True` should do the job